from_cpython/CMakeLists.txt

```diff
@@ -158,21 +156,7 @@ add_library(FROM_CPYTHON OBJECT ${STDMODULE_SRCS} ${STDOBJECT_SRCS} ${STDPYTHON_
 add_dependencies(FROM_CPYTHON copy_stdlib)
 
 set(STDMODULES
-    ${CMAKE_BINARY_DIR}/lib/python2.7/lib-dynload/bz2.pyston.so
```

Comment:
First of all, this is minor, but let's try to always copy from v2.7.7 of CPython.  There aren't that many changes between the micro versions so it's not that huge of a deal, but it helps us if we have a convention around it (for example if we try to find any remaining Pyston changes).

I think changing setup.py is perfectly fine if we don't support all of it yet.  My only recommendation would be to try to minimize the size of the diff so that it's more understandable (1500 changed lines is quite a lot).  For example, if we are commenting out the calls to certain functions, we probably don't need to comment out the functions themselves.  And if we don't support some feature such as multi-platform, instead of commenting out all of that code we could maybe just hard-code the platform.